src/cloudflare/internal/vectorize-api.ts

```diff
@@ -7,14 +7,15 @@ interface Fetcher {
   fetch: typeof fetch;
 }
 
-enum Operation {
-  INDEX_GET = 0,
-  VECTOR_QUERY = 1,
-  VECTOR_INSERT = 2,
-  VECTOR_UPSERT = 3,
-  VECTOR_GET = 4,
-  VECTOR_DELETE = 5,
-}
+const Operation = {
+  INDEX_GET: 'INDEX_GET' as const,
```

Comment:
Nit: These look more verbose than needed, but if the linters somehow prefer that that's fine

Reply:
erasableSyntaxOnly disallows typescript only types such as enum. Unfortunately, this is the javascript equivalent of the previous enum declaration.